library/core/src/mem/mod.rs

```diff
@@ -1113,7 +1113,10 @@ impl<T> fmt::Debug for Discriminant<T> {
 /// # Stability
 ///
 /// The discriminant of an enum variant may change if the enum definition changes. A discriminant
-/// of some variant will not change between compilations with the same compiler.
+/// of some variant will not change between compilations with the same compiler. See the [Reference]
+/// for more information.
+///
+/// [Reference]: ../../reference/items/enumerations.html#discriminants
```

Comment:
This section doesn't exist until https://github.com/rust-lang/reference/pull/1055 is merged, but [the linked page](https://doc.rust-lang.org/nightly/reference/items/enumerations.html#discriminants) still contains some relevant information.

Reply:
The CI didn't like this, so we need to revert 946d51e8ba2501aa24256317d8798e475dc862b5 later.